peer.go

```diff
@@ -40,8 +40,8 @@ const (
 	retryReadWriteMessageAttempts        = 5
 	reconnectInterval                    = 10 * time.Second
 
-	pingInterval                   = 2 * time.Minute
-	connectionHealthTickerDuration = 3 * time.Minute
+	pingIntervalDefault                   = 30 * time.Second
+	connectionHealthTickerDurationDefault = 1 * time.Minute
```

Comment:
Now that you implemented this option `WithPingInterval()`, maybe we can stick to the previous defaults, so 2 minutes for `pingInterval` and 3 minutes for `connectionHealthTickerDuration`, because these are the defaults that nodes are using I believe (although I'm not a 100% sure about the `connectionHealthTickerDuration`), and change these values to 30 seconds and 1 minute in Arc with this new option. What do you think?